{
  "config": {
    "step": {
      "user": {
        "title": "Zimi",
        "description": "You can find the MAC on the back of your Zimi Cloud Connect.",
        "data": {
          "host": "[%key:common::config_flow::data::host%]",
          "mac": "MAC"
        },
        "data_description": {
          "mac": "You can find the MAC on the back of your Zimi Cloud Connect."
        }
      }
    },
    "error": {
      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
      "unknown": "[%key:common::config_flow::error::unknown%]",
      "timeout": "Connection timed out]",
      "invalid_host": "Invalid host name",
      "connection_refused": "Connection refused",
      "discovery_failure": "ZCC discovery via UDP failed."
    },
    "abort": {
      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
    }
  }
}
